android/build.gradle

```diff
@@ -6,14 +6,14 @@
  */
 
 buildscript {
-	ext.kotlin_version = '1.3.61'
+	ext.kotlin_version = '1.3.72'
 
 	repositories {
 		google()
 		jcenter()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:3.6.2'
+		classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
It looks like `4.0.1` is available

Reply:
Google literally released 4.0.1 yesterday. :sweat_smile: 
https://mvnrepository.com/artifact/com.android.tools.build/gradle/4.0.1

I'll update it now. Thanks.